html/webappapis/scripting/processing-model-2/compile-error-same-origin-with-hash.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+ <head>
+  <title>window.onerror - compile error in &lt;script src=...> with hash</title>
```

Comment:
Sorry for my delayed reply. I thought it's more consistent with "&lt;", but as you commented, it's not necessary. I'm ok with leaving it as is.